### ISSUE TITLE
Playhead for the pattern editor

### ISSUE THREAD

**Issue by spectromas** [LMMS/lmms#1195]
I'm sure this must have been mentioned before as it is such an oddity to me (as a new user) but I can't find anything mentioned about it here.

Firstly there is the beats and bassline editor. The main thing with this is that there is no way to tell what is playing in the pattern or where you are. If I create a 32 bar pattern in it I have to blindly guess where to put things which seems really odd and quite time consuming (not to mention difficult in some instances). Why is there not something to show you where the play head is? A simple step-like light above or below the pattern would suffice.

**Comment by Csini** [comment]
please implement this feature

**Comment by Monospace-V** [comment]
Same. Being able to move this playhead around, I feel, is important too.
![image](https://user-images.githubusercontent.com/76674645/195590064-f0612435-40e7-4cff-8348-6ac46a06f622.png)

I'm trying to listen to something for reference and editing the second half of this. It's frustrating to have to listen to the entirety of the first half for something like this.

**Comment by YesImAsh** [comment]
Why hasn't it been added? it sounds like a genuinely good idea. It makes it easier to edit the B + B editor, and you can have not just 1 bar, BUT 2 BARS! Without the need the need to stretch the entire thing to the point where you can barely see anything! and its easy as that. I dont know why this hasn't been added, but one thing for sure, this is a must have.

**Comment by Rossmaxx** [comment]
I agree this is a must have. We just don't got anyone to code it. The existing devs are busy with their own stuff. If you can, feel free to code it yourself. We are here to help.

**Comment by regulus79** [comment]
hiii
Maybe #7794 can help <3

**Comment by Rossmaxx** [comment]
@regulus79 good catch. That is what the user is asking for.
